index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+<header>
+    <nav class="nav-bar" >
+        <div class="logo"><span>John Doe </span> </div>
+        <div class="menu"><span><img src="/img/Union.png" alt="menu"> </span> </div>
```

Comment:
- [ ] Let's change the `div` tag to a `nav` tag. Remember the Hamburg icon is holding all the menu actions.

Reply:
Thanks but I don't understand exactly what I needed to do as suggested, do you mean I should have another nav bar on the icon Instead of a div, remember a nav tag is wrapping all of them together already